Changelog.md

```diff
@@ -7,6 +7,8 @@ Breaking changes:
  * JSON AST: Remove members with ``null`` value from JSON output.
  * Parser: Disallow ``gwei`` as identifier.
  * Parser: Disallow dot syntax for ``value`` and ``gas``.
+ * Parser: Disallow non-printable characters in string literals.
+ * Parser: Introduce Unicode string literals: ``unicode"😃"``.
```

Comment:
This is actually not a breaking change, but we can also keep it here.

Reply:
The way it is implemented is, because it uses `unicode` as a keyword.